q.cpp

```diff
@@ -0,0 +1,245 @@
+#include <stdio.h>
```

Comment:
ok, so, let's remove the file then.
It is impossible to forget about the function, it will be needed shortly ...